src/tools/rbd_mirror/PoolReplayer.cc

```diff
@@ -453,6 +454,10 @@ int PoolReplayer<I>::init_rados(const std::string &cluster_name,
 			        const std::string &description,
 			        RadosRef *rados_ref,
                                 bool strip_cluster_overrides) {
+  dout(10) << "cluster_name=" << cluster_name << ", client_name=" << client_name
+	   << ", mon_host=" << mon_host << ", strip_cluster_overrides="
+	   << strip_cluster_overrides << dendl;
```

Comment:
nit: vim shows above two lines has a combination of spaces and tabs.

Reply:
I needed spaces to line them up . Can remove them if that is a problem.

Reply:
It's better to stick to spaces and avoid tabs entirely.  We aren't consistent about it though so I'm merging as is.